.github/workflows/containers.yaml

```diff
@@ -0,0 +1,157 @@
+    if: |
+      github.ref != 'refs/heads/master' &&
+      contains(needs.changed-files.outputs.all_modified_files, ',dockerfiles/cli-standalone/Dockerfile')
+    with:
+      workload-id-provider: projects/1094498259535/locations/global/workloadIdentityPools/gh-celo-monorepo/providers/github-by-repos
+      service-account: 'celo-monorepo-dev@devopsre.iam.gserviceaccount.com'
+      artifact-registry: us-west1-docker.pkg.dev/devopsre/dev-images/celocli
+      tags: testing
+      context: .
+      file: dockerfiles/cli-standalone/Dockerfile
+      trivy: true
+  celocli-build:
+    uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.13.1
+    needs: changed-files
+    name: Build us-west1-docker.pkg.dev/devopsre/celo-monorepo/celocli:latest
+    if: |
+      github.ref == 'refs/heads/master' &&
+      contains(needs.changed-files.outputs.all_modified_files, ',dockerfiles/cli-standalone/Dockerfile')
+    with:
+      workload-id-provider: projects/1094498259535/locations/global/workloadIdentityPools/gh-celo-monorepo-master/providers/github-by-repos
+      service-account: 'celo-monorepo@devopsre.iam.gserviceaccount.com'
+      artifact-registry: us-west1-docker.pkg.dev/devopsre/celo-monorepo/celocli
+      tags: latest
+      context: .
+      file: dockerfiles/cli-standalone/Dockerfile
+      trivy: true
+
+  # Blockscout Metadata crawler images
+  metadata-crawler-build-dev:
+    uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.13.1
+    needs: changed-files
+    name: Build us-west1-docker.pkg.dev/devopsre/dev-images/blockscoute-metadata-crawler:testing
+    if: |
+      github.ref != 'refs/heads/master' &&
+      contains(needs.changed-files.outputs.all_modified_files, ',dockerfiles/metadata-crawler')
+    with:
+      workload-id-provider: projects/1094498259535/locations/global/workloadIdentityPools/gh-celo-monorepo/providers/github-by-repos
+      service-account: 'celo-monorepo-dev@devopsre.iam.gserviceaccount.com'
+      artifact-registry: us-west1-docker.pkg.dev/devopsre/dev-images/blockscoute-metadata-crawler
+      tags: testing
+      context: .
+      file: dockerfiles/metadata-crawler/Dockerfile
+      trivy: true
+  metadata-crawler-build:
+    uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.13.1
+    needs: changed-files
+    name: Build us-west1-docker.pkg.dev/devopsre/dev-images/blockscoute-metadata-crawler:latest
+    if: |
+      github.ref == 'refs/heads/master' &&
+      contains(needs.changed-files.outputs.all_modified_files, ',dockerfiles/metadata-crawler')
+    with:
+      workload-id-provider: projects/1094498259535/locations/global/workloadIdentityPools/gh-celo-monorepo/providers/github-by-repos
+      service-account: 'celo-monorepo-dev@devopsre.iam.gserviceaccount.com'
+      artifact-registry: us-west1-docker.pkg.dev/devopsre/dev-images/blockscoute-metadata-crawler
```

Comment:
Ditto about `blockscoute`. Also, is the "prod" (`main` branch) image also supposed to go to `dev-images`??

Reply:
Thanks for the typo.

The registry I thought that for very "minor" images maybe the effort of adding a new registry does not worth it. But maybe is better to follow same patter everywhere.